source/process/release-metrics-definitions.rst

```diff
@@ -0,0 +1,136 @@
+| Metric                                  | How to Measure                                                               |
++=========================================+==============================================================================+
+| Time updated to RC1 (PST)               | Check Release Discussion channel history                                     |
+|                                         | for date/time RC1 was cut.                                                   |
++-----------------------------------------+------------------------------------------------------------------------------+
+| How many RCs cut                        | Check Release Discussion channel history                                     |
+|                                         | for how many RCs were cut for that release.                                  |
++-----------------------------------------+------------------------------------------------------------------------------+
+| Number of days between when final RC    | Check Release Discussion channel for post with official release build.       |
+| is cut and the release date             | Oxygen = 16th - Day Final RC is cut                                          |
++-----------------------------------------+------------------------------------------------------------------------------+
+| Community + customer bugs reported      | Use "Customer-bug" and "Community-bug" label for release month, from         |
+| during release timeframe (17th to 16th) | 17th to 16th (from previous release day to current release day).             |
++-----------------------------------------+------------------------------------------------------------------------------+
+| Number of customer bugs fixed           | "Customer-bug" tickets closed and marked for current release.                |
+| during release                          |                                                                              |
++-----------------------------------------+------------------------------------------------------------------------------+
+| Total valid bugs in fix version         | After closing current release:                                               |
+|                                         |                                                                              |
+|                                         | project = Mattermost AND issuetype = Bug AND resolution not in (Duplicate,   |                                         
+|                                         | "Cannot Reproduce", "Won't Fix") AND fixVersion = latestReleasedVersion()    |
++-----------------------------------------+------------------------------------------------------------------------------+
+| Total valid bugs in fix version found   | Check "Se", "Selenium-found, "Rainforest-found" Jira labels.                 |
```

Comment:
> It doesn't have a filter. One of the remaining gaps is to consider which filters are most accurate.

That's okay, in that I would propose a more complete Jira query for now that you use today to measure it, like the one used for `Total valid bugs in fix version` below, and we can iterate over time